code/modules/clothing/under/accessories/accessory_vr.dm

```diff
@@ -27,6 +27,21 @@
 	icon_state = "collar_bell"
 	item_state = "collar_bell_overlay"
 	overlay_state = "collar_bell_overlay"
+	var/nospam = 0
+	
+/obj/item/clothing/accessory/collar/bell/verb/jinglebell()
+	set name = "Jingle Bell"
+	set category = "Object"
+	set src in usr
+	if(!istype(usr, /mob/living)) return
+	if(usr.stat) return
+	
+	if(!nospam)
+		usr.audible_message("[usr] jingles the [src]'s bell.")
+		nospam = 1
+		sleep(30)
+		nospam = 0
```

Comment:
`schedule_callback_in(3 SECONDS, VARSET_CALLBACK(src, nospam, 0))`

Starts a timer to reset the var to 0 after 3 seconds. Also "If not nospam" reads weird. Double negatives are bad. Maybe rename the var to "played_recently" or "cooldown" or something.

Reply:
Doesn't seem to work in testing.  The var stays at 1, and an error is printed to the server log.
```
[2018-11-12T12:49:53] Runtime in scheduler.dm,143: Cannot execute null.Invoke().
  Caught by process: scheduler
  proc name: process (/datum/scheduled_task/callback/process)
  src: /datum/scheduled_task/callback (/datum/scheduled_task/callback)
  src.loc: /datum/scheduled_task/callback (/datum/scheduled_task/callback)```